### PR TITLE
Make some env var fixes for collectors

### PR DESCRIPTION
**Pull request by JDragovichAlertLogic**

### CODE REVIEW

File: cfn/guardduty.template [hunk 1]
```diff
@@ -16,7 +16,8 @@
           "Type": "String",
           "Default": "api.global-services.global.alertlogic.com",
           "AllowedValues" : [
-               "api.global-services.global.alertlogic.com"
+               "api.global-services.global.alertlogic.com",
+               "api.global-integration.product.dev.alertlogic.com"
```
Comment (kkuzmin) [inline cfn/guardduty.template]: not sure we need this.

File: index.js [hunk 2]
```diff
@@ -152,7 +152,11 @@ exports.handler = function(event, context) {
             // Migration code for old collectors.
             // This needs to be done because the collector lambda does not have premissions to set its own env vars.
             if(!process.env.aws_lambda_update_config_name){
-                process.env.aws_lambda_update_config_name = 'configs/lambda/config.json';
+                process.env.aws_lambda_update_config_name = 'configs/lambda/al-cwe-collector.json';
+            }
+            //add in the env var for the framework
+            if(!process.env.stack_name && event.StackName){
+                process.env.stack_name = event.StackName;
```
Comment (kkuzmin) [inline index.js]: Let's separate this code into a function and call it even before `async.waterfall` is called.

File: cfn/guardduty.template (hunk 1 above)
Comment (JDragovichAlertLogic) [reply cfn/guardduty.template]: Sorry. that slipped in.

File: index.js [hunk 3]
```diff
@@ -144,20 +144,27 @@ function getStatisticsFunctions(event) {
...
-            // This needs to be done because the collector lambda does not have premissions to set its own env vars.
-            if(!process.env.aws_lambda_update_config_name){
-                process.env.aws_lambda_update_config_name = 'configs/lambda/config.json';
-            }
 
-            const collector = new AlAwsCollector(
-                context,
-                "cwe",
+exports.handler = function(event, context) {
+envVarMigration(event);
```
Comment (kkuzmin) [inline index.js]: please fix the indention.

**Pull request merged**